src/Appwrite/Migration/Version/V22.php

```diff
@@ -145,17 +145,17 @@ private function migrateCollections(): void
                     $this->dbForProject->purgeCachedCollection($id);
                     break;
                 case 'memberships':
-                    $indexes = [
-                        '_key_roles',
-                    ];
-                    foreach ($indexes as $index) {
-                        try {
-                            $this->createIndexFromCollection($this->dbForProject, $id, $index);
-                        } catch (Throwable $th) {
-                            Console::warning("Failed to create index \"$index\" from {$id}: {$th->getMessage()}");
-                        }
-                    }
-                    $this->dbForProject->purgeCachedCollection($id);
+//                    $indexes = [
+//                        '_key_roles',
+//                    ];
+//                    foreach ($indexes as $index) {
+//                        try {
+//                            $this->createIndexFromCollection($this->dbForProject, $id, $index);
+//                        } catch (Throwable $th) {
+//                            Console::warning("Failed to create index \"$index\" from {$id}: {$th->getMessage()}");
+//                        }
+//                    }
+//                    $this->dbForProject->purgeCachedCollection($id);
```

Comment:
We'll need to check if it exists here and drop it, to remove it for self-hosted instances that it got added to